Composer/packages/form-dialogs/src/components/FormDialogPropertiesEditor.tsx

```diff
@@ -78,6 +80,7 @@ export const FormDialogPropertiesEditor = React.memo((props: Props) => {
 
   const schemaIdRef = React.useRef<string>(schema.id);
 
+  const { undo, redo, canUndo, canRedo } = useUndo();
```

Comment:
You could return undo as undefined when you can't undo but maybe having the booleans is semantically clearer.